tutorials/current-place-on-map/current-place-on-map/MapViewController.swift

```diff
@@ -46,43 +68,51 @@ class ViewController: UIViewController {
                                           longitude: defaultLocation.coordinate.longitude,
                                           zoom: zoomLevel)
     mapView = GMSMapView.map(withFrame: view.bounds, camera: camera)
-    mapView.isMyLocationEnabled = true
     mapView.settings.myLocationButton = true
     mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+    //mapView.isMyLocationEnabled = true
```

Comment:
I didn't like the look of the map with location enabled, but I wanted to leave that there to indicate that it can be done.